spec/api_spec.rb

```diff
@@ -35,13 +42,16 @@
   end
 
   describe "GET /interviewers" do
-    let(:collection) { [double(value: interviewer1, key: '001'), double(value: interviewer2, key: '002')] }
-    let(:interviewer1) { {name: 'Jichao', email: 'jichao@thoughtworks.com', key: '001'} }
-    let(:interviewer2) { {name: 'Ouyang', email: 'ouayng@thoughtworks.com', key: '002'} }
+    let(:collection) { [double(value: interviewer1_db, key: '001'), double(value: interviewer2_db, key: '002')] }
+    let(:interviewer1_db) { {"name"=> 'Jichao', "email"=> 'jichao@thoughtworks.com', "work_from"=> 2011} }
+    let(:interviewer2_db) { {"name"=> 'Ouyang', "email"=> 'ouayng@thoughtworks.com', "work_from"=> 2015} }
+
+    let(:interviewer1_modle) { {name: 'Jichao', email: 'jichao@thoughtworks.com', key: '001', work_from: 2011, experience: 4} }
```

Comment:
Line is too long. [127/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.